dlrs/main/classes/RollupSummariesSelector.cls

```diff
@@ -34,6 +34,32 @@ public class RollupSummariesSelector {
   @TestVisible
   private CustomMetadataSelector m_customMetadataSelector;
 
+  private static Map<String, List<RollupSummary>> summariesCacheByKey = new Map<String, List<RollupSummary>>();
+
+  private List<RollupSummary> allSummaries;
+
+  public static void clearRollupCache() {
+    summariesCacheByKey = new Map<String, List<RollupSummary>>();
+  }
+
+  @TestVisible
+  private static void setRollupCache(
```

Comment:
Useful data mocking if needed. Will be very useful when we want to refactor out all the tests that add Custom Objects to use Custom Metadata while being able to ignore what might already exist in the org.